### PR TITLE
Ch  css bugs

### PR DESCRIPTION
**Pull request by Ariuka11**
## Description
Getting rid of all the px values in the LESS file
## Type of change
Please delete options that are not relevant.
- [x] Bug fix (non-breaking change which fixes an issue)

## Checklist
Remove any items which are not applicable.
- [x] I have performed a self-review of my own code
## Link to Trello card
https://trello.com/c/jwpguhm4/93-css-bugs

### CODE REVIEW

**Review by Cireimu — DISMISSED** [review]
Unsure of the specific goal of this PR, as there are still px values throughout the less values. Another thing is that the text looks to be extremely small on everything now. However, if the goal was to just remove px values on the specific text then I think you did the job, just need to make some Trello cards noting the need of adding proper sizing to everything again. Also, it looks like navbar links in the admin view are hidden, looking in the developer tools they are still there, just that they are hidden for some reason.

With these things in mind I am willing to push it through, but only on the condition that the sizing will be corrected ASAP. Before we can merge this PR I will need you to also pull from developing and resolve any and all merge conflicts that are blocking this PR from merging.

**Review by bkandaris — COMMENTED** [review]
I'm thinking; you fixed the text sizing on another branch? I've reviewed a few PRs of yours and one or two of them the sizing looks great and on others it's still small?

**Pull request merged**
